h/static/scripts/controllers/search-bar-controller.js

```diff
@@ -0,0 +1,139 @@
+'use strict';
+
+var Controller = require('../base/controller');
+var setElementState = require('../util/dom').setElementState;
+
+/**
+ * Controller for the search bar.
+ */
+class SearchBarController extends Controller {
+  constructor(element) {
+    super(element);
+
+    this._input = element.querySelector('.js-search-bar-input');
+    this._dropdown = element.querySelector('.js-search-bar-dropdown');
+    this._dropdownItems = element.querySelectorAll('.js-search-bar-dropdown-item');
+
+    var self = this;
+    var index = -1;
+    var menuLen = this._dropdownItems.length;
+
+    var updateActiveDropdownItem = function (index) {
+      clearActiveDropdownItem();
+      self._dropdownItems[index].classList.add('js-search-bar-dropdown-menu-item--active');
+    };
+
+    var clearActiveDropdownItem = function () {
+      if (element.querySelector('.js-search-bar-dropdown-menu-item--active')) {
+        element.querySelector('.js-search-bar-dropdown-menu-item--active').
```

Comment:
Can you avoid calling `querySelector()` twice by saving its result in a local variable?